_posts/2014-04-02-using-modules-and-the-resolver.md

```diff
@@ -108,7 +108,7 @@ under `app/helpers`. If your custom helper contains a dash(`upper-case`,
 // app/helpers/upper-case.js
 import Ember from "ember";
 
-export default Ember.Handlebars.makeBoundHelper(function(value, options) {
+export default Ember.Helper.helper(function(value, options) {
```

Comment:
The signature here is different.  For `Ember.Helper.helper` the callback function accepts two args (params and hash).  I think this needs to be tweaked:

```
export default Ember.Helper.helper(function([value]) {
```